livekit-agents/livekit/agents/pipeline/room_io.py

```diff
@@ -28,288 +33,189 @@ class RoomInputOptions:
     """Sample rate of the input audio in Hz"""
     audio_num_channels: int = 1
     """Number of audio channels"""
-    audio_queue_capacity: int = 0
+    text_input_topic: str | None = TOPIC_CHAT
+    """Topic for text input"""
+    _audio_queue_capacity: int = 0
     """Capacity of the internal audio queue, 0 means unlimited"""
-    video_queue_capacity: int = 0
+    _video_queue_capacity: int = 0
     """Capacity of the internal video queue, 0 means unlimited"""
-    forward_user_transcript: bool = True
-    """Whether to forward user transcript segments to the room"""
 
 
 @dataclass(frozen=True)
 class RoomOutputOptions:
```

Comment:
I think some of the options aren't relevant for the users, but nit for now, let's merge